src/core/thread/mle.cpp

```diff
@@ -637,6 +641,17 @@ void Mle::SetAssignLinkQuality(const Mac::ExtAddress aMacAddr, uint8_t aLinkQual
     }
 }
 
+uint8_t Mle::GetRouterSelectionJitter(void) const
+{
+    return mSetRouterJitter ? mRouterJitter : mRouterSelectionJitterTimeout;
+}
+
+void Mle::SetRouterSelectionJitter(uint8_t aRouterJitter)
+{
+    mSetRouterJitter = true;
```

Comment:
I don't think we need this `bool`, just initialize `mRouterJitter` to `kRouterSelectionJitter` and set it directly.